app/styleguide/mock-data/cCartPost.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  totals: {
+    subtotal: 100,
```

Comment:
- [ ] Please increase to 100000 so the number groubs become visible (e.g `1'000.00)